### DISCUSSION TITLE
How do I get the status box to go away?

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#2063] [Q&A] [answered]
When the status box shows up, how do I get it to go away? I see that the arrow keys work, but if you've got something that doesn't require me to then correct the state, that would be preferable.

**Comment by saulpw** [comment, marked as answer]
Does `Ctrl+L` (refresh-screen) work for this case?

Also note that `b` will toggle the sticky (non-status) sidebar.

**Reply by cool-RR** [reply]
It does, thank you.
